backend/package-lock.json

```diff
@@ -950,6 +950,14 @@
       "integrity": "sha512-GsVpkFPlycH7/fRR7Dhcmnoii54gV1nz7y4CWyeFS14N+JVBBhY+r8amRHE4BwSYal7BPTDp8isvAlCxyFt3Hg==",
       "dev": true
     },
+    "csv-parser": {
```

Comment:
I think this `csv-parser` was for the csv import function which we implement on another repo, right? So the package-lock prolly shouldn't need to be updated here